containers/performance_tests/TestDynRankView.hpp

```diff
@@ -113,7 +113,7 @@ struct InitDynRankViewFunctor {
   void operator()(const int i) const {
     for (unsigned j = 0; j < _inview.extent(1); ++j) {
       for (unsigned k = 0; k < _inview.extent(2); ++k) {
-        _inview(i, j, k) = i / 2 - j * j + k / 3;
+        _inview(i, j, k) = double(i) / 2 - j * j + double(k) / 3;
```

Comment:
Changes result but if tests pass I don't care.

Reply:
This is a performance test and we don't check results. On the other, this might or might not have impact on the benchmark timings.